src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -881,31 +881,36 @@ class WorkspaceService(protected val userInfo: UserInfo, dataSource: DataSource,
   def listSubmissions(workspaceName: WorkspaceName): Future[PerRequestMessage] =
     dataSource.inFutureTransaction(readLocks=Set(workspaceName)) { txn =>
       withWorkspaceContextAndPermissions(workspaceName, WorkspaceAccessLevels.Read, txn) { workspaceContext =>
-        Future.successful(RequestComplete(StatusCodes.OK, containerDAO.submissionDAO.list(workspaceContext, txn).toList))
+        val statuses = containerDAO.submissionDAO.list(workspaceContext, txn) map { submission =>
+          withRawlsUser(submission.submitter, txn) { user =>
+            new SubmissionStatusResponse(submission, user)
+          }
+        }
+        Future.successful(RequestComplete(StatusCodes.OK, statuses.toList))
       }
     }
 
-  def createSubmission(workspaceName: WorkspaceName, submissionRequest: SubmissionRequest): Future[PerRequestMessage] =
-    withSubmissionParameters(workspaceName,submissionRequest) {
+  def createSubmission(workspaceName: WorkspaceName, submissionRequest: SubmissionRequest): Future[PerRequestMessage] = {
```

Comment:
this change is trickier: I wanted to launch all the workflows and commit all of them to the db before starting the monitors. This is somewhat in prep for handling the power going out. This way if there is an error getting credentials or otherwise starting the monitor, at least we have recorded that the workflows were started.